packages/admin/src/api/index.ts

```diff
@@ -0,0 +1,516 @@
+  const req = new pb.ListOrgsRequest()
+  const res: pb.ListOrgsResponse = await api.unary(
+    APIService.ListOrgs,
+    req,
+    ctx,
+  )
+  return res.getListList().map(pbToOrgInfo)
+}
+
+/**
+ * Removes the current org.
+ * @param ctx Context containing gRPC headers and settings.
+ * These will be merged with any internal ctx.
+ * @internal
+ */
+export async function removeOrg(
+  api: GrpcConnection,
+  ctx?: ContextInterface,
+): Promise<void> {
+  logger.debug('remove org request')
+  const req = new pb.RemoveOrgRequest()
+  await api.unary(APIService.RemoveOrg, req, ctx)
+  return
+}
+
+/**
+ * Invites the given email to an org.
+ * @param email The email to add to an org.
+ * @param ctx Context containing gRPC headers and settings.
+ * These will be merged with any internal ctx.
+ * @internal
+ */
+export async function inviteToOrg(
+  api: GrpcConnection,
+  email: string,
+  ctx?: ContextInterface,
+): Promise<string> {
+  logger.debug('invite to org request')
+  const req = new pb.InviteToOrgRequest()
+  req.setEmail(email)
+  const res: pb.InviteToOrgResponse = await api.unary(
+    APIService.InviteToOrg,
+    req,
+    ctx,
+  )
+  return res.getToken()
+}
+
+/**
+ * Removes the current session dev from an org.
+ * @param ctx Context containing gRPC headers and settings.
+ * These will be merged with any internal ctx.
+ * @internal
+ */
+export async function leaveOrg(
+  api: GrpcConnection,
+  ctx?: ContextInterface,
+): Promise<void> {
+  logger.debug('leave org request')
+  const req = new pb.LeaveOrgRequest()
+  await api.unary(APIService.LeaveOrg, req, ctx)
+  return
+}
+
+/**
+ * (Re-)enables billing for an account, enabling usage beyond the free quotas.
+ * @param ctx Context containing gRPC headers and settings.
+ * These will be merged with any internal ctx.
+ * @internal
+ */
+export async function setupBilling(
+  api: GrpcConnection,
+  ctx?: ContextInterface,
+): Promise<void> {
+  logger.debug('setup billing request')
+  const req = new pb.SetupBillingRequest()
+  await api.unary(APIService.SetupBilling, req, ctx)
+  return
+}
+
+/**
+ * Returns a billing portal session url.
+ * @param ctx Context containing gRPC headers and settings.
+ * These will be merged with any internal ctx.
+ * @internal
+ */
+export async function getBillingSession(
+  api: GrpcConnection,
+  ctx?: ContextInterface,
+): Promise<string> {
+  logger.debug('get billing session request')
+  const req = new pb.GetBillingSessionRequest()
+  const res: pb.GetBillingSessionResponse = await api.unary(
+    APIService.GetBillingSession,
+    req,
+    ctx,
+  )
+  return res.getUrl()
+}
+
+export interface Usage {
+  description: string
+  units: number
+  total: number
+  free: number
+  grace: number
+  cost: number
+  period?: Period
+}
+
+export interface Period {
+  unixStart: number
+  unixEnd: number
+}
+
+export interface Customer {
+  // TODO: Have to properly unmarshall this object rather than use toObject
```

Comment:
This todo might be relevant when we actually start using these apis and creating tests for them. toObject returns base64 encoded strings, which is usually not what we want.